React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -33,7 +33,7 @@ @implementation RCTTextInputComponentView {
 #if !TARGET_OS_OSX // [macOS]
   RCTUIView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
 #else // [macOS
-  RCTUITextView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
+  RCTPlatformView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
```

Comment:
This should be `RCTUITextView`, however our `RCTUITextView` is not typedef'd to a `NSView` like core does w/ `UIView`.

Will need to fix our shim to work here.